store/tikv/mocktikv/cop_handler_dag.go

```diff
@@ -104,6 +108,38 @@ func (h *rpcHandler) handleCopDAGRequest(req *coprocessor.Request) *coprocessor.
 	return buildResp(chunks, counts, err)
 }
 
+func (h *rpcHandler) handleCopStream(req *coprocessor.Request) (tikvpb.Tikv_CoprocessorStreamClient, error) {
+	if len(req.Ranges) == 0 {
+		return nil, errors.New("request range is null")
+	}
+	if req.GetTp() != kv.ReqTypeDAG {
+		return nil, errors.Errorf("unsupported request type %d", req.GetTp())
+	}
+
+	dagReq := new(tipb.DAGRequest)
+	err := proto.Unmarshal(req.Data, dagReq)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	sc := flagsToStatementContext(dagReq.Flags)
+	timeZone := time.FixedZone("UTC", int(dagReq.TimeZoneOffset))
+	ctx := &dagContext{
+		dagReq:    dagReq,
+		keyRanges: req.Ranges,
+		evalCtx:   &evalContext{sc: sc, timeZone: timeZone},
+	}
+	e, err := h.buildDAG(ctx, dagReq.Executors)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
```

Comment:
Can line112-135 be extracted into a function? It seems the same as the function of `handleCopDAGRequest`.